pkg/operator/ceph/csi/spec.go

```diff
@@ -199,6 +202,11 @@ func StartCSIDrivers(namespace string, clientset kubernetes.Interface, ver *vers
 	tp.RBDGRPCMetricsPort = getPortFromENV("CSI_RBD_GRPC_METRICS_PORT", DefaultRBDGRPCMerticsPort)
 	tp.RBDLivenessMetricsPort = getPortFromENV("CSI_RBD_LIVENESS_METRICS_PORT", DefaultRBDLivenessMerticsPort)
 
+	enableSnap := os.Getenv("CSI_RBD_ENABLE_SNAPSHOTTER")
+
+	if !strings.EqualFold(enableSnap, "false") {
+		tp.EnableSnapshotter = "true"
```

Comment:
Shouldn't this be decoupled into both rbd and cephfs?

Reply:
the main reason to disable the snapshot is due to the alpha feature, this holds good for both rbd and cephfs 
updated the env variable name to support both cephfs and rbd

Reply:
> the main reason to disable the snapshot is due to the alpha feature, this holds good for both rbd and cephfs


> updated the env variable name to support both cephfs and rbd

Where is this?

Reply:
Oh ok, now we have a more general variable. Sounds good!